examples/quoted-identifiers/quoted_identifiers.bal

```diff
@@ -1,10 +1,22 @@
 import ballerina/io;
 
-// Use the reserved keyword `function` as the name of the function.
+// Use the reserved keyword `function` as the function identifier.
```

Comment:
Was there a particular reason to change this? IMO, it's clearer to say name.

Should we at least change this to
```suggestion
// Use the reserved keyword `function` as the identifier of the function.
```